website/docs/patterns/data-visualization/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,47 @@
+## Deciding how best to represent data
```

Comment:
@didoo Do you think this section is necessary/helpful to include? I was thinking of reaching out for input from the Data Viz Working Group to flesh it out.